### PR TITLE
Changed image tag to image digest.

### PR DESCRIPTION
**Pull request by jonquilwilliams**
https://bugzilla.redhat.com/show_bug.cgi?id=1878477

Needs QE & Peer review.

Will need CP to 4.2 onward through 4.6

### CODE REVIEW

**Comment by ahardin-rh** [comment]
LGTM! :tada: I omitted a label for 4.2, as we no longer actively support docs for 4.2.

File: modules/images-configuration-registry-mirror.adoc [hunk 1]
```diff
@@ -191,12 +191,12 @@ unqualified-search-registries = ["registry.access.redhat.com", "docker.io"]
...
 
-.. Pull an image to the node from the source and check if it is actually
-resolved by the mirror.
+.. Pull an image digest to the node from the source and check if it is actually
+resolved by the mirror. The `ImageContentSourcePolicy` supports image digests only, not image tags.
 +
 [source,terminal]
 ----
-sh-4.2# podman pull --log-level=debug registry.access.redhat.com/ubi8/ubi-minimal
+sh-4.2# podman pull --log-level=debug registry.access.redhat.com/ubi8/ubi-minimal@sha256:5cfbaf45ca96806917830c183e9f37df2e913b187adb32e89fd83fa455ebaa6
```
Comment (xiuwang) [inline modules/images-configuration-registry-mirror.adoc:199]: In the step 1 , the image used `registry.access.redhat.com/ubi8/ubi-minimal@sha256:c505667389712dc337986e29ffcb65116879ef27629dc3ce6e1b17727c06e78f` , could you update above to make consistent?
 `skopeo copy \
docker://registry.access.redhat.com/ubi8/ubi-minimal@sha256:c505667389712dc337986e29ffcb65116879ef27629dc3ce6e1b17727c06e78f \
docker://example.io/example/ubi-minimal`

File: modules/images-configuration-registry-mirror.adoc [hunk 2]
```diff
@@ -191,12 +191,12 @@ unqualified-search-registries = ["registry.access.redhat.com", "docker.io"]
     insecure = false
```
Comment (xiuwang) [inline modules/images-configuration-registry-mirror.adoc:191]: In step 5d, could you help update the cmd `sh-4.2# cat /etc/containers/registries` to `sh-4.2# cat /etc/containers/registries.conf`

File: modules/images-configuration-registry-mirror.adoc (hunk 1 above)
Comment (jonquilwilliams) [reply modules/images-configuration-registry-mirror.adoc:199]: Thia change has been made and pushed to this branch. Please verify that it is correct.

File: modules/images-configuration-registry-mirror.adoc (hunk 2 above)
Comment (jonquilwilliams) [reply modules/images-configuration-registry-mirror.adoc:191]: This change has been made and pushed to this branch. Please verify that it is correct.

**Comment by jonquilwilliams** [comment]
@xiuwang I have made your changes. Please verify. Thanks!

**Comment by xiuwang** [comment]
LGTM, thanks

**Pull request merged**